### ISSUE TITLE
Slicer with data validation causes corruption

### ISSUE THREAD

**Issue by wa-nelson** [EPPlusSoftware/EPPlus#842]
We've recently upgraded from v6.1.3 to v6.2.1 and are now generating corrupt files when we add a slicer to a worksheet which has data validation.

Reproducible sample:
``` c#
using ExcelPackage package = new();
ExcelWorksheet ws = package.Workbook.Worksheets.Add("WS1");

ws.SetValue("A1", "0");
ws.SetValue("A2", "0");
ws.SetValue("A3", "0");

var address = new ExcelAddress(1, 1, 3, 1);
IExcelDataValidationInt dataValidation = ws.DataValidations.AddIntegerValidation(address.Address);
dataValidation.Formula.Value = 0;
dataValidation.Formula2.Value = 1;

ExcelRange range = ws.Cells[1, 1, 3, 1];
ExcelTable table = ws.Tables.Add(range, "Table");
table.StyleName = "None";

ExcelTableSlicer slicer = ws.Drawings.AddTableSlicer(table.Columns[0]);

File.WriteAllBytes(_savePath, package.GetAsByteArray());
```

The file written by the code above will warn of a corrupt spreadsheet when we open it, but if we remove the slicer (or data validation) the spreadsheet opens without a problem.

**Comment by JanKallman** [comment]
Thanks, we will have a look at this and get back to you

**Comment by OssianEPPlus** [comment]
Fixed as of 6.2.2
